02-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -417,11 +420,11 @@ on our data.
 Create a map of the North Eastern United States as follows:
 
 1. Import and plot `Boundary-US-State-NEast.shp`. Adjust line width as necessary.
-2. REPROJECT the layer into UTM zone 18 north.
+2. *Reproject* the layer into UTM zone 18 north.
 3. Layer the Fisher Tower point location `point_HARV` on top of the above plot.
 4. Add a title to your plot.
-5. Add a LEGEND to your plot that shows both the state boundary (line) and the 
-Tower location POINT.  
+5. Add a legend to your plot that shows both the state boundary (line) and the 
```

Comment:
This again goes to my previous question of style guide. It is something (else) we should discuss if we don't have an established one to follow.  
The area I'd like to see more consistency in is in the addition of emphasis.  Currently we use ALL CAPS, **bold**, and *italics* interchangeably.    
I too was inconsistent in my changes.  If consistent, it would be "**Reproject** the layer..", "Add a **title** to the...", and "Add a **legend** to...".  That to me seems like too much bold considering they are also a numbered list were each item is a single number on the list. 

Here's my take:
1) I'm not a fan of all caps for emphasis and reserve all caps for abbreviations.  All caps are too often interpreted as the reader being yelled at. 
2) I prefer bold for less formal writing when a new topic is introduced and it needs to be easily seen when scanning back to look for the topic again. (In formal writing, bold is reserved for headings).  
3) I prefer italics for all other emphasis.  When one is reading through and needs to have a concept, etc emphasized but is something that is unlikely to be referred back to.  Differences in closely related directions is a great use of this.